arbnode/transaction_streamer.go

```diff
@@ -866,7 +869,11 @@ func (s *TransactionStreamer) executeNextMsg(ctx context.Context, exec *executio
 	}
 	err = s.exec.DigestMessage(pos, msg)
 	if err != nil {
-		log.Info("feedOneMsg failed to send message to execEngine", "err", err, "pos", pos)
+		logger := log.Warn
+		if prevMessageCount < msgCount {
+			logger = log.Debug
+		}
```

Comment:
a different option would be to only debug if we are the main sequencer, but this approach seems more resilient.